src/main/webapp/app/exam/manage/students/exam-students.component.html

```diff
@@ -24,6 +24,25 @@ <h3 class="mb-0">
                 <fa-icon [icon]="'plus'" [fixedWidth]="true"></fa-icon>&nbsp;
                 <span jhiTranslate="artemisApp.examManagement.examStudents.registerAllFromCourse"></span>
             </button>
+            <button
```

Comment:
I think this button should get a similar size as the other buttons.
![image](https://user-images.githubusercontent.com/33342534/110961207-d12d6400-834f-11eb-9bc1-9704702c723a.png)

Reply:
Thanks for catching that! Done 👍